app/models/user.rb

```diff
@@ -66,7 +74,20 @@ def password_reset_expired?
   end
 
   def feed
-    Micropost.where("user_id = ?", id)
+    Micropost.where("user_id IN (?) OR user_id = ?",
```

Comment:
`where("user_id IN (?) OR user_id = ?",  following_ids, id).by_created_at` viết scope cho code này

Reply:
dạ rồi ạ